### PR TITLE
Migrate V5 documentation to production

### PR DESCRIPTION
**Pull request by zguesmi**

### CODE REVIEW

**Review by james-toussaint — APPROVED** [review]
:+1:

File: for-developers/confidential-computing/create-your-first-sgx-app.md [hunk 1]
```diff
@@ -89,7 +84,7 @@ That should be enough for this tutorial, but if you have other specifications yo
...
+Well that's where the file `protect-fs.sh` comes in place. If you inspect the content of this script, you can see that we use the famous [fspf](intel-sgx-technology.md#fspf-file-system-protection-file) feature of SCONE. We use SCONE's [CLI](https://sconedocs.github.io/SCONE_CLI/) to authenticate the file system directories that can be used by the application \(/bin, /lib...\) as well as the code itself, and take a snapshot of their state. This snapshot will be later shared with the enclave \(via the Blockchain\) to make sure everything is under control. If we change one bit of one of the authenticated files, the file system's state changes completely and the enclave will refuse to boot since it is a possible attack.
```
Comment (ericr6) [inline for-developers/confidential-computing/create-your-first-sgx-app.md]: famous ! to remove?

File: for-developers/your-first-app.md [hunk 2]
```diff
@@ -36,38 +36,38 @@ iExec leverage [Docker](https://www.docker.com/why-docker) containers to ensure
...
 
-Input files contain non sensitive data publicly available on the Internet. The requester may specify any number of input files in the requestorder.
+Input files contain non-sensitive data publicly available on the Internet. The requester may specify any number of input files in the requestorder.
 
 For each input file, the variable `IEXEC_INPUT_FILE_NAME_x` is set to the file name \(`x` is the index of the file starting with `1`\). The total number of input files is stored in the variable.
 
 Use these variables in your application to find input files to process. \(first input file path is `/iexec_in/$IEXEC_INPUT_FILE_NAME_1`\)
 
-#### Datasets \(confidential input files\):
+#### Confidential input files \(datasets\):
```
Comment (ericr6) [inline for-developers/your-first-app.md:66]: confidential datafiles ?

**Review by ericr6 — APPROVED** [review]

File: for-developers/your-first-app.md (hunk 2 above)
Comment (zguesmi) [reply for-developers/your-first-app.md:66]: I, personally, prefer not to add more iExec-specific vocabulary (datafile) to what already exists. What do you think?

File: for-developers/confidential-computing/create-your-first-sgx-app.md (hunk 1 above)
Comment (zguesmi) [reply for-developers/confidential-computing/create-your-first-sgx-app.md]: Yes, I'll remove it thanks!

**Pull request merged**